addons/mail/static/src/composer/composer.js

```diff
@@ -384,6 +384,18 @@ export class Composer extends Component {
         return ev.isTrusted;
     }
 
+    /**
+     * @param {import('@mail/core/message_model').Message} message
+     */
+    _notifyMailbox(message) {
+        if (this.props.composer.thread.type === "mailbox") {
+            this.env.services.notification.add(
+                sprintf(_t('Message posted on "%s"'), message.originThread.displayName),
+                { type: "info" }
+            );
+        }
+    }
```

Comment:
The problem is that you only want to send the gif URL and not the composed message.
If someone write something, add attachment or what ever and decide to send a gif, it will also send the composed message.
I agree that `_notifyMailbox` feel strange, but I don't know what else to do. `sendMessage` does too much stuff.

Reply:
`sendMessage()` consumes current state of composer model.
Would be nice to call this same function without consuming the composer model state, but you pass exactly what you want.

This would solve this code duplication by slightly increasing the complexity of this function. Conceptually this would stay quite simple, so it's the better trade-off IMO.

Reply:
I get it now, should be good now :)